test/Npgsql.PluginTests/Npgsql.PluginTests.csproj

```diff
@@ -1,12 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
-
   <PropertyGroup>
-    <TargetFrameworks>net452;netcoreapp2.2</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT' OR '$(CoreOnly)' == 'True'">netcoreapp2.2</TargetFrameworks>
-    <DebugType>portable</DebugType>
-    <AssemblyOriginatorKeyFile>../../Npgsql.snk</AssemblyOriginatorKeyFile>
+    <SignAssembly>false</SignAssembly>
```

Comment:
Any reason to prevent signing this specifically?

Reply:
Because the GeoJSON plugin is unsigned, and it uses the unsigned GeoJSON library.

Reply:
So let's hope we'll never need to have `[InternalsVisibleTo]` to the plugin tests project... :)